threshsign/src/ThresholdSignaturesTypes.cpp

```diff
@@ -230,7 +230,7 @@ void Cryptosystem::validatePrivateKey(const std::string& key) const {
 }
 
 bool Cryptosystem::isValidCryptosystemSelection(const std::string& type, const std::string& subtype) {
-  UNUSED(subtype);
+  (void)subtype;
```

Comment:
remove UNUSED(subtype);
In general, I would remoe this macro compeltely with C++17. maybe we should open a task if you see it being used so many times?
`bool Cryptosystem::isValidCryptosystemSelection(const std::string& type, [[maybe_unused]] const std::string& subtype) {
`

I won't comment more about UNUSED, please fix all occurences by using   [[maybe_unused]]

Reply:
Done.